QSAR-ready_SMILES_pipeline/QSAR_ready_workflow-test.yml

```diff
@@ -0,0 +1,10 @@
+- doc: QSAR_ready_SMILES_workflow_test
+  job:
+    input_data1:
+      class: File
+      path: "https://github.com/RECETOX/workflow-test-data/raw/main/QSAR-ready_SMILES_pipeline/pubchem_reduced_failedInChIFiltered.inchi"
+  outputs:
+    outfile_final:
+      file: "https://github.com/RECETOX/workflow-test-data/raw/main/QSAR-ready_SMILES_pipeline/final_table.smi"
```

Comment:
@trachtok output file should not be a link (Looks like link is not yet supported by galaxy workflow testing and results in `FileNotFoundError`). So, output file need to be present in a directory where workflow file *.ga is present or in any sub-directory and need to be referred accordingly.

Reply:
Thanks for pointing that out, I have added the output file and changed .yml file.

Reply:
@trachtok thanks for the changes.